tests/utils/Statistic/interpolate-data.test.ts

```diff
@@ -0,0 +1,105 @@
+import { describe, expect, test } from 'vitest'
+import interpolateData from '$lib/utils/Statistic/interpolate-data'
+import data from "./interpolate-data-test.json" assert { type: 'json' };
```

Comment:
let's stick to using .js files instead of .ts

Reply:
Will change it

Reply:
On second thought, since this function has defined types for parameters, I think we should leave the file as .ts to test for parameter's types. Wdyt?

Reply:
can you give an example testing those parameter types?

Reply:
I don't have any test for those yet